tests/e2e/integration/networks.js

```diff
@@ -84,6 +85,7 @@ describe('Test cases for networks page', () => {
       .should('contain', defaultNetwork.middlewareUrl)
       .goBack()
       .goBack()
+      .wait(2500)
       .get('.transaction-list')
       .should('exist')
```

Comment:
I can't get why `wait` is needed here.
> Create an assertion. Assertions are automatically retried until they pass or time out.
https://docs.cypress.io/api/commands/should#Syntax

Reply:
The goal of this PR was to find the weak spots, and prove them by restarting test with empty commits. That's why there were status/hodl label, this was not the final stage of development (intentionally). You are right it's is better to increase timeout for this get.